js/views/pageNavVw.js

```diff
@@ -617,19 +798,19 @@ module.exports = baseVw.extend({
   },
 
   mouseenterServerSubmenuTrigger: function() {
-    this.serverSubmenu.css('right', this.$('.popMenu-navBarSubMenu-pageNav').outerWidth());
+    this.$serverSubmenu.css('right', this.$('.popMenu-navBarSubMenu-pageNav').outerWidth());
 
-    this.ServerSubmenuTimeout = setTimeout(() => {
-      this.serverSubmenu.addClass('server-submenu-opened');
+    this.$serverSubmenuTimeout = setTimeout(() => {
```

Comment:
I'm totally for prefacing cached jQuery elements with a $, but this is an integer representing a timeout (i.e. not a jQuery object), so I don't think we should be prefacing it with the dollar sign.